x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/rest/SecurityRestFilter.java

```diff
@@ -88,6 +95,11 @@ public void handleRequest(RestRequest request, RestChannel channel, NodeClient c
                         e -> handleException("Secondary authentication", request, channel, e)));
                 }, e -> handleException("Authentication", request, channel, e)));
         } else {
+            if (request.getHeaders() != null && request.getHeaders().containsKey("Authorization")) {
```

Comment:
Do we want this? I would have thought we'd keep the warning behaviour from #70114

Reply:
Right. I need to revisit this PR after I merge #701114